tests/Assembly/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -20,6 +38,30 @@ contract Assembly {
       }
     }
   }
+
+  function caseAssembly() {
+    assembly {
+      switch value
+      case 0 {
+        mstore(0, 0x0000000000000000000000000000000000000000000000000000000000000000)
+      }
+      case 1 {
+        mstore(0, 0x1111111111111111111111111111111111111111111111111111111111111111)
+      }
+    }
+  }
+
+  function forAssembly() {
+    assembly {
+      for {
```

Comment:
I do agree with you, this seems a little bit odd but that's a good starting point; I'll wait for you to address #92 and then we can merge both